stl/inc/limits

```diff
@@ -1133,14 +1133,23 @@ _NODISCARD constexpr int _Countr_zero(const _Ty _Val) noexcept {
     return _Countr_zero_fallback(_Val);
 }
 
+_NODISCARD _CONSTEXPR20 inline int _Get_isa_available() noexcept {

```

Comment:
This could be unconditionally `constexpr`

Reply:
No, it can't. In pre-C++20, when `is_constant_evaluated()` is not available, we don't have compile-time fallback, and always return `__isa_available`, so it is never constexpr.